x-pack/filebeat/module/cisco/shared/ingest/asa-ftd-pipeline.yml

```diff
@@ -383,7 +383,7 @@ processors:
       if: "ctx._temp_.cisco.message_id == '304001'"
       field: "event.outcome"
       description: "304001"
-      value: success
+      value: "allowed"
```

Comment:
I don't see _allowed_ or _dropped_ specified as valid values for event.outcome, only _success_ and _failure._ My understanding is that this should go into event.type. Am I missing something?

Reply:
These are temporary values that are used to communicate to the painless later what the combination of `outcome` and type `should` be. If state is squashed to a binary of success/failure it is not possible to reconstruct the possibility of a successful denial. I could have done if with a second variable, but this feels better.

Reply:
These get sorted out here https://github.com/elastic/beats/pull/29698/commits/76fe29f3f6e543c25b81f7308690c772697691b7#diff-35840a6c4edae3a4fc97e0a1c84d5576fcfd9bb64faf12e7a6800347a5728cb1R1857-R1872. If you'd like I can add commentary explaining this.